bundle/edu.gemini.pot/src/test/scala/edu/gemini/pot/sp/ProgramGen.scala

```diff
@@ -220,7 +235,7 @@ object ProgramGen {
 
   val genEdit: Gen[ProgEdit] =
     oneOf(
-      genEditDataObject,
+      frequency(7 -> genEditDataObject, 3 -> genEditObsPhase2Status),
```

Comment:
Yes, exactly.  I think in reality it's nowhere near 30% phase 2 status changes but since it has such a big impact on permissions I wanted to make it prevalent.